db/fixtures/aws_instance_types.yml

```diff
@@ -1,220 +1,233 @@
 ---
 a1.2xlarge:
   :architecture:
-  - :x86_64
-  :cluster_networking: 
+  - :arm64
+  :cluster_networking:
   :current_generation: true
   :current_version: true
-  :description: A1 General purpose 2XL
-  :ebs_only: true
-  :ebs_optimized_available: true
+  :description: A1 Double Extra Large
+  :ebs_optimized_available: false
   :enhanced_networking: true
   :family: General purpose
   :instance_store_size: 0
   :instance_store_size_gb: 0.0
-  :instance_store_type: 
+  :instance_store_type:
   :instance_store_volumes: 0
   :intel_aes_ni: true
-  :intel_avx: 
-  :intel_avx2: 
-  :intel_turbo: 
+  :intel_avx:
+  :intel_avx2:
+  :intel_turbo:
   :memory: 17179869184
   :memory_gb: 16.0
   :name: a1.2xlarge
-  :network_performance: :up_to_10_gigabit
+  :network_performance: :very_high
```

Comment:
This was a legit bug in the current yaml, the network performance string is "Up To 10 Gigabit" which didn't match our exact "10 Gigabit" or "20 Gigabit" check